test/integration/standard-governor/propose/standardGovernorPropose.t.sol

```diff
@@ -6,14 +6,10 @@ import { IERC20 } from "../../../../lib/common/src/interfaces/IERC20.sol";
 
 import { IERC5805 } from "../../../../src/abstract/interfaces/IERC5805.sol";
 
-import {
-    IntegrationBaseSetup,
-    IBatchGovernor,
-    IGovernor,
-    IPowerToken,
-    IRegistrar,
-    IStandardGovernor
-} from "../../IntegrationBaseSetup.t.sol";
+import { IGovernor } from "../../../../src/abstract/interfaces/IGovernor.sol";
+import { IBatchGovernor } from "../../../../src/abstract/interfaces/IBatchGovernor.sol";
+
+import { IntegrationBaseSetup, IPowerToken, IRegistrar, IStandardGovernor } from "../../IntegrationBaseSetup.t.sol";
```

Comment:
import `IPowerToken`, `IRegistrar`, `IStandardGovernor` directly from their locations instead?

Reply:
Fixed the import paths in this commit: https://github.com/MZero-Labs/ttg/pull/263/commits/6cf80ea42e617e75069247b0dfbb9cfa871ecb80